homeassistant/components/reolink/entity.py

```diff
@@ -156,3 +159,34 @@ async def async_will_remove_from_hass(self) -> None:
             self._host.async_unregister_update_cmd(cmd_key, self._channel)
 
         await super().async_will_remove_from_hass()
+
+
+class ReolinkChimeCoordinatorEntity(ReolinkChannelCoordinatorEntity):
+    """Parent class for Reolink chime entities connected."""
+
+    def __init__(
+        self,
+        reolink_data: ReolinkData,
+        chime: Chime,
+        coordinator: DataUpdateCoordinator[None] | None = None,
```

Comment:
Why can the coordinator be None?

Reply:
Because when it is None, the default coordinator will be used (almost all reolink entities use that coordinator).
See https://github.com/home-assistant/core/blob/94c0b9fc069b02bab31dff04e71ae0457e03f336/homeassistant/components/reolink/entity.py#L53-L54